### PR TITLE
[FRONT-1457] Services to interact with Operators and Sponsorships

### PR DESCRIPTION
**Pull request by tumppi**
Added fully typed Graph client. Also added basic services to interact with Operators and Sponsorships. Also added a quick and dirty Sponsorships listing to showcase how to use the client.

In order to generate the `src/gql.js` (source for all your typed queries) run
```
npm run codegen
```

It will connect to the specified GraphQL endpoint and introspect schema + queries defined in code with the `gql` template.

### CODE REVIEW

File: .graphclientrc.yml
Comment (tumppi) [inline .graphclientrc.yml:1]: This is a new addition which is used to generate fully typed The Graph client for us. We need to sort out how we provide subgraph address here dynamically.

File: app/src/services/operators.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+import { Address } from '$shared/types/web3-types'
+
+const getOperatorChainId = () => {
+    // TODO: add to .toml
+    const sponsorshipChainName = 'dev1'
+    const chainConfig = getConfigForChainByName(sponsorshipChainName)
+    return chainConfig.id
+}
+
+gql`
```
Comment (pmichalowski) [inline app/src/services/operators.ts:19]: why this is not being assigned to any const/var? what does it actually do then?

Comment (tumppi) [reply app/src/services/operators.ts:19]: `npm run build-graph-client` command goes through these `gql` templates and generates typed functions for said queries. Then it's available as a function through `getBuiltGraphSDK()`.

**Comment by tumppi** [comment]
@mondoreale @pmichalowski: Feel free to merge this when you need this stuff. It should not affect anything on master.

**Comment by mondoreale** [comment]
Taking over! FYI @pmichalowski

File: .graphclientrc.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+sources:
+  - name: streamr
+    handler:
+      graphql:
+        endpoint: ${NETWORK_SUBGRAPHS_ENDPOINT:http://localhost:8000}/subgraphs/name/streamr-dev/network-subgraphs
```
[the remark this answers is not in the document]
Comment (mondoreale) [reply .graphclientrc.yml:5]: I'm still looking for a better way though.

[the remark this answers is not in the document]
Comment (mondoreale) [reply .graphclientrc.yml:5]: Reason? If you have a different docker setup then you'll also generate slightly different ts files. Btw, I think I got something.. less abstract.

File: app/src/services/operators.ts (hunk 1 above)
Comment (mondoreale) [reply app/src/services/operators.ts:18]: It's `npm run codegen` now and it generates a single file. The mechanics stays the same.

**Comment by mondoreale** [comment]
The GQL endpoint is now defined using `GQL_SUBGRAPH_PATH` env variable. It defaults to the `network-subgraphs` in the default docker stack.

**Pull request merged**